test/runtests.jl

```diff
@@ -904,11 +1085,12 @@ end
     m2 = lm(x, y, dropcollinear=false)
 
     p1 = predict(m1, x, interval=:confidence)
-    p2 = predict(m2, x, interval=:confidence)
+    #predict uses chol hence removed
```

Comment:
This should be kept for now as Cholesky is still the default. But add tests to check that these throw an error when QR is used.

Reply:
Thanks. I will revise it.